src/convertToObject.js

```diff
@@ -12,7 +12,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const object = {};
+  let properties = sourceString.split(';').map(property => property.trim());
+
+  properties = properties.filter(property => property.length > 0);
```

Comment:
You can chain several methods.

```suggestion
  let properties = sourceString
    .split(';')
    .map(property => property.trim())
    .filter(property => property.length > 0);
```